container/home/builder/.default_aliases

```diff
@@ -88,12 +88,12 @@ alias ctp-reset="ansible-playbook /srv/inventories/start.yml -e=deploy_mode=rese
 
 ### MISC ###
 
-alias ctp-update-os="ansible-playbook /srv/inventories/start.yml -e=deploy_mode=deploy-role -e=single_role=nova.core.updates -l"
```

Comment:
Also add ` -e role_only=true` after deploy mode. This will skip some of the tasks and make the command faster.

Reply:
Here okay as its single_role